lib/tasks/dev.rake

```diff
@@ -9,73 +9,64 @@ task sample_data: :environment do
   Rating.delete_all
   User.delete_all
 
-  people = Array.new(10) do
+  people = Array.new(30) do
     {
-      first_name: Faker::Name.first_name,
-      last_name: Faker::Name.last_name
+      username_generated_from_name: Faker::Name.unique.first_name
     }
   end
 
-  people << { first_name: 'Alice', last_name: 'Smith' }
-  people << { first_name: 'Bob', last_name: 'Smith' }
-  people << { first_name: 'Carol', last_name: 'Smith' }
-  people << { first_name: 'Doug', last_name: 'Smith' }
+  people << { username_generated_from_name: 'Alice' }
+  people << { username_generated_from_name: 'Bob' }
+  people << { username_generated_from_name: 'Carol' }
+  people << { username_generated_from_name: 'Doug' }
 
   people.each do |person|
-    username = person.fetch(:first_name).downcase
+    username = person.fetch(:username_generated_from_name).downcase
 
     user = User.create(
       email: "#{username}@example.com",
       password: 'password',
-      username: username.downcase,
-      name: "#{person[:first_name]} #{person[:last_name]}",
-      private: [true, false].sample
+      username: username,
+      customer: [true, true, false].sample
     )
 
     p user.errors.full_messages
   end
 
   users = User.all
 
-  users.each do |first_user|
-    users.each do |second_user|
-      if rand < 0.75
-        first_user_follow_request = first_user.sent_follow_requests.create(
-          recipient: second_user,
-          status: FollowRequest.statuses.values.sample
-        )
-
-        p first_user_follow_request.errors.full_messages
-      end
+  users.where({ :customer => true }).each do |complaining_customer|
+    users.where({ :customer => false }).each do |customer_representative|
+      next unless rand < 0.4
 
-      next unless rand < 0.75
+      complaining_customer_grade_submission = complaining_customer.gradee_ratings.create(
+        gradee: customer_representative,
+        grader: complaining_customer,
+        grade: rand(5)
+      )
 
-      second_user_follow_request = second_user.sent_follow_requests.create(
-        recipient: first_user,
-        status: FollowRequest.statuses.values.sample
+      customer_representative_grade_submission = customer_representative.gradee_ratings.create(
+        gradee: complaining_customer,
+        grader: customer_representative,
+        grade: rand(5)
       )
 
-      p second_user_follow_request.errors.full_messages
-    end
-  end
+      pick_a_status = %i[open closed urgent].sample
```

Comment:
Since `Conversation#status` is a Enum column, you can get this dynamically with `Conversations.statuses.keys`. You can read more about enums [here](https://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html).

Reply:
Thank you for the tip! I ended up using `Conversation.statuses.keys.sample` to generate the random values